src/app_manager/AppManagerDeployer.js

```diff
@@ -1,5 +1,6 @@
 import Logger from '../utils/Logger'
 import AppManagerWrapper from './AppManagerWrapper'
+import ContractsProvider from '../utils/ContractsProvider'
```

Comment:
What I discussed with @facuspagnuolo is that the ContractsProvider is expected to be a global object, so I'm afraid that this could break the AppManager for CLI. If the complex example can be made to work without this line, then I'd remove it and review it when dealing with https://github.com/zeppelinos/zos-lib/issues/83.